src/components/Genome3D/List/index.js

```diff
@@ -157,6 +157,10 @@ export const List = (
               </Link>
             )}
             isSearchable={true}
+            customiseSearch={{
+              validation: /^([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2})$/i,
```

Comment:
It seems that genome3d is still not supporting case insensitive, maybe we should `toUpperCase` in the URL when you are searching by this field.